utils/socket_connect.py

```diff
@@ -0,0 +1,37 @@
+import socket
+
+
+def open_socket(host='127.0.0.1', port=50000):
```

Comment:
Need to add constants for host and port, smth like DEFAULT_HOST and DEFAULT_PORT